app/helpers/curation_concerns/file_set_helper.rb

```diff
@@ -7,9 +7,12 @@ def parent_path(parent)
     end
   end
 
-  def media_display(file_set, locals = {})
-    render media_display_partial(file_set),
-           locals.merge(file_set: file_set)
+  # REVIEW: Since this media display could theoretically work for
```

Comment:
Thoughts, @tpendragon @escowles @jcoyne?

I'm inclined to say :shipit: for now.